common/unsafe/src/main/java/org/apache/spark/sql/catalyst/util/CollationAwareUTF8String.java

```diff
@@ -439,7 +450,7 @@ public static UTF8String toTitleCase(final UTF8String target) {
     return UTF8String.fromString(toTitleCase(target.toString()));
   }
 
-  public static String toTitleCase(final String target) {
+  private static String toTitleCase(final String target) {
```

Comment:
Same here: accept and return UTF8String values?

Reply:
Do we actually need this method? Couldn't we just implement everything inside the public method above?

Reply:
discussing in: https://github.com/apache/spark/pull/47014#discussion_r1652818328